nva-datamodel-java/src/main/resources/publication-ontology.ttl

```diff
@@ -1196,6 +1196,9 @@ nva:VfxSupervisor a rdfs:Class .
 nva:VideoEditor a rdfs:Class .
 nva:WorkPackageLeader a rdfs:Class .
 nva:Writer a rdfs:Class .
+nva:TextualOrBoolean a rdfs:Class ;
+  rdfs:label "Textual or Boolean"@en ;
+  rdfs:comment "Superclass for textual and boolean values"@en .
```

Comment:
Not sure if this is needed?

Reply:
We should accept publisherVersion as a boolean, but it should only be outputted as a string.

Yes, it's weird (and temporary until we have done our batch scan)

Reply:
It probably isn't. If you can get away without, do.

Reply:
Pushed to mavenLocal, publication-api seems fine without the TextualOrBoolean class.